src/Facebook/InstantArticles/Elements/Blockquote.php

```diff
@@ -83,10 +83,25 @@ public function toDOMElement($document = null)
         if (!$document) {
             $document = new \DOMDocument();
         }
+
+        if (!$this->isValid()) {
+            return $this->emptyElement($document);
+        }
+
         $element = $document->createElement('blockquote');
 
         $element->appendChild($this->textToDOMDocumentFragment($document));
 
         return $element;
     }
+
+    /**
+     * Overrides the @see TextContainer::isValid().
+     *
+     * @return true for valid Blockquote that contains text, false otherwise.
+     */
+    public function isValid()
+    {
+        return !Type::isTextEmpty($this->text) || parent::isValid();
```

Comment:
Oh, wait! I think `$this->text` is not used on rendering, this is leftover from a refactoring as far as I can see. We need to remove this property, and remove the also broken `withText` method. In that case, you don't need to override the `isValid` method.